inc/functions/files.php

```diff
@@ -1458,6 +1459,12 @@ function _rocket_get_cache_dirs( $url_host, $cache_path = '', $hard_reset = fals
 		$cache_path = rocket_get_constant( 'WP_ROCKET_CACHE_PATH' );
 	}
 
+	// When Windows-based.
+	$is_windows = ( DIRECTORY_SEPARATOR === '\\' && ( substr( $cache_path, 0, 7 ) !== 'vfs://' ) );
```

Comment:
7 is the length, but we compare with a string that is 6 characters, am I missing something?

Reply:
Nope. Mistake. Fixed in https://github.com/wp-media/wp-rocket/pull/2632/commits/e4335ecf48ce9c2be744b4557b4c32558a492612.